github/resource_github_repository_test.go

```diff
@@ -377,6 +405,9 @@ func testAccCheckGithubRepositoryAttributes(repo *github.Repository, want *testA
 		if *repo.HasDownloads != want.HasDownloads {
 			return fmt.Errorf("got has downloads %#v; want %#v", *repo.HasDownloads, want.HasDownloads)
 		}
+		if &repo.Topics != &want.Topics {
```

Comment:
You will probably need to rewrite this comparison somewhat, as the default value for a slice is `nil` but its equivalent to an empty slice, so you'll want to check len and iterate:

```go
if len(want.Topics) != len(repo.Topics) {
			return fmt.Errorf("got has topics %#v; want %#v", repo.Topics, want.Topics)
		}
		for i := range want.Topics {
			if repo.Topics[i] != want.Topics[i] {
				return fmt.Errorf("got has topics %#v; want %#v", repo.Topics, want.Topics)
			}
		}
```